src/__test__/components/data-processing/configure-embedding/CalculationConfig.test.jsx

```diff
@@ -62,8 +62,6 @@ describe('Data Processing CalculationConfig', () => {
 
   afterEach(() => {
     jest.clearAllMocks();
-    mockOnConfigChange.mockClear();
-    mockOnPipelineRun.mockClear();
```

Comment:
jest.clearAllMocks is sufficient for cleaning up all the mocked function call history